platform/kubernetes/kubelet/client.go

```diff
@@ -138,23 +136,6 @@ func (c *client) GetPodStats(ctx context.Context) (*kubeletTypes.PodStats, error
 	return stats, nil
 }
 
-// GetPodStats gets pod spec
-func (c *client) GetSpec(ctx context.Context) (*cadvisorTypes.MachineInfo, error) {
-	req, err := c.newRequest(specPath)
```

Comment:
[nit] `specPath` is now unused

Reply:
! removed at https://github.com/mackerelio/mackerel-container-agent/pull/80/commits/4c8a6eedd2389e2017d9469212ec1195fa93230a